### PR TITLE
Render all student

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- Handle render all student

### CODE REVIEW

File: practices/practice-one/src/scripts/controllers/student.controller.js [hunk 1]
```diff
@@ -3,6 +3,29 @@ import { axiosClient } from '../helpers/utils.js';
 export default class Controller {
     constructor() {}
 
+    async getStudents() {
```
Comment (cunguyendev) [inline practices/practice-one/src/scripts/controllers/student.controller.js:6]: Can we move this as a helper then re-use?

Comment (DuongPham-dev) [reply practices/practice-one/src/scripts/controllers/student.controller.js:6]: I don't know yet, but my intention is to use it to get all students to use in render

File: practices/practice-one/src/scripts/views/student-item.view.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            style="background-image: url(${this.#image});"></div>
+            <p class="student-item-name">${this.#name}</p>`;
+        this.#liElement.addEventListener('click', (e) => {
+            this.#handleViewProfile();
+        });
+        return this.#liElement;
+    }
+
+    #handleViewProfile(id) {
+        console.log(id);
```
Comment (cunguyendev) [inline practices/practice-one/src/scripts/views/student-item.view.js]: Remove console log

File: practices/practice-one/src/scripts/views/student-item.view.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        this.#liElement.classList.add('student-item');
+        this.#id = id;
+        this.#name = name;
+        this.#image =
+            'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQFIjfHSc0yfN71WXEzJBNJOH-cs1GvoVSYOg&usqp=CAU';
+    }
+
+    createElement() {
+        this.#liElement.setAttribute('id', this.#id);
+        this.#liElement.innerHTML = `<div class="student-item-image"
```
Comment (cunguyendev) [inline practices/practice-one/src/scripts/views/student-item.view.js:18]: Apply template

**Pull request merged**